src/web_app_skeleton/script/setup.ecr

```diff
@@ -34,6 +34,12 @@ if [ -z "$CI" ]; then
 fi
 printf "✔ Done\n" | indent
 
+printf "\n▸ Checking for a .env\n"
```

Comment:
I think instead of checking we could print something in the `if` like "Created a .env file"`. That way if you already have one it doesn't say anything. 

But just a suggestion. I'm ok with it as-is if you feel that's better!